content/apps/managing-teammates.md

```diff
@@ -0,0 +1,24 @@
+---
+menu:
+  main:
+    parent: advanced
+title: Managing Teammates
+weight: 10
+---
+
+To add a new teammate to your organization:
+
+1. [Send them an invite](https://login.cloud.gov/invitations/new).
+1. Run
+
+    ```bash
+    cf set-org-role <email> <org> OrgManager
+    ```
+
+1. Add them to each space:
+
+    ```bash
+    cf set-space-role <email> <org> <space> SpaceDeveloper
```

Comment:
Should this be `SpaceManager` instead? Both? Is there an easier way to accomplish all of the role assignment, e.g. through the deck?